hermes/nullprobe/proto/nullprobe.proto

```diff
@@ -0,0 +1,32 @@
+// Licensed under the Apache License, Version 2.0 (the "License")
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License. 
+ 
+// Author: Alicja Kwiecinska (kwiecinskaa@google.com) github: alicjakwie 
+
+syntax = "proto2";
+
+import "github.com/google/cloudprober/probes/proto/config.proto";
+import "github.com/googleinterns/step224-2020/config/proto/probe.proto";
+option go_package = "github.com/googleinterns/step224-2020/hermes/nullprobe/proto";
+
+// The hermes.nullprobe package encapsulates the test probe, which creates files in GCS with randomized contents.
+package hermes.nullprobe;
+
+message ProbeConf {
+  optional string bucket = 1; // bucket name
```

Comment:
Nit: no need to comment on self-describing fields. In general: the comment should contain more information. If it's just repeating what the code says, it can be removed.